src/main/java/org/apache/solr/benchmarks/beans/Cluster.java

```diff
@@ -54,5 +54,8 @@ public static class ExternalSolrConfig {
 
     @JsonProperty("ssh-username")
     public String sshUserName;
+
+    @JsonProperty("protocol")
+    public String protocol = "http";
```

Comment:
Makes sense for now. I'm thinking that we should convert this and others like this to Enums for better validation and readability. Maybe in a separate refactoring effort.